fapolicy_analyzer/ui/trust_file_list.py

```diff
@@ -160,32 +175,37 @@ def process_rows(queue, total):
                 return False
 
             count = self._get_tree_count()
-            if count < total:
+
+            if count < self.total:
                 pct = int(count / total * 100)
```

Comment:
May sure whatever we decide for the total you are using the same total variable here as you do in the if check on line 179.

Reply:
We still end up needing to read from the thread because we only know the initial total number of entries. I am seeing smoother behavior but its still pretty quick at the end. instead of 10% to done its, 10%, 50%, 80%, done.